semanticdb/semanticdb3/src/main/scala/scala/meta/internal/semanticdb3/Scala.scala

```diff
@@ -22,6 +23,31 @@ object Scala {
     def isEmptyPackage: Boolean = symbol == Symbols.EmptyPackage
     def isGlobal: Boolean = !isNone && Descriptor.descriptorLasts.contains(symbol.last)
```

Comment:
This should be updated to take multi symbols into account.

Reply:
You mean by adding `!isMulti && ...`? I don't think we should do `asMulti.exists(_.isGlobal)`

Reply:
Done, made it
```scala
    def isGlobal: Boolean =
      !isNone &&
        Descriptor.descriptorLasts.contains(symbol.last) &&
        (!isMulti || asMulti.exists(_.isGlobal))
    def isLocal: Boolean =
      !isNone &&
        !isGlobal &&
        (!isMulti || asMulti.exists(_.isLocal))
```